src/main.ts

```diff
@@ -18,7 +18,7 @@ async function bootstrap() {
       .build()
     
     const document = SwaggerModule.createDocument(app, options)
-    SwaggerModule.setup('api-hub', app, document)
+    SwaggerModule.setup('zl-id', app, document)
```

Comment:
name it 'zapi-identity' please

Reply:
ok sir.. sorry i cant join the meeting.. i'll effect the change ASAP